en/getting_started/README.md

```diff
@@ -11,11 +11,11 @@ It is [dual-licensed under Apache 2.0 and GPLv3](https://github.com/mavlink/qgro
 To get the source files:
 1. Clone the repo (or your fork):
    ```
-   git clone --recursive https://github.com/mavlink/qgroundcontrol.git
+   git clone https://github.com/mavlink/qgroundcontrol.git
    ```
-2. Update submodules (do this each time you pull new source code):
+2. Update submodules (required each time you pull new source code):
    ```
-   git submodule update
+   git submodule update --init --recursive 
```

Comment:
FYI, I believe you need `--recursive` on every update.

Reply:
I think recursive is needed if the submodules are nested. I've never used it myself. I always just do and init and then an update.

Reply:
That is correct. The recursive line was in the `clone` line above previously so I assumed that it was needed (?). If not, it should just be removed.

Reply:
OK, I bet this is just the easiest way to do an --init so you don't have to specified it in the next line. I will revert.

Reply:
Correct on the --init.